pandas/core/config_init.py

```diff
@@ -249,6 +249,27 @@
     method. Valid values: False,True
 """
 
+pc_latex_multicolumn = """
+: bool
+    This specifies if the to_latex method of a Dataframe uses multicolumns
+    to pretty-print MultiIndex columns.
+    method. Valid values: False,True
```

Comment:
I think this 'method. ' here is a left-over from copying it from somewhere else?